pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>hpi</packaging>
     <name>Jenkins IvyTrigger Plug-in</name>
     <version>0.36-SNAPSHOT</version>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/IvyTrigger+Plugin</url>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/IvyTrigger+Plugin</url>
```

Comment:
The Jenkins Wiki is effectively defunct and may not be around for much longer.  Perhaps best to point this to the git repo?  We could add a README.md to the root of the repo with the wiki information (and usage instructions) for the purpose...

Reply:
Thanks for the hint. There is even a guideline for it https://jenkins.io/doc/developer/publishing/wiki-page/. I have to say that I am not a big fan of this ongoing migration as it means a lot of effort and will result in a situation where many plugins will be using documentation in their repository and many plugins still using the wiki. Therefore I'd ask you to perform the migration.